packages/styles/scss/components/_statcard.scss

```diff
@@ -17,6 +17,10 @@
 
       @include cornercut(72px, 40px);
 
+      [dir="rtl"] & {
+        @include cornercut(72px, 40px, right);
```

Comment:
Pretty sure this should be 
```scss
@include cornercut(72px, 40px, left);
```

Reply:
Sorry about this @justintemps will make the change :)